src/main/resources/templates/in-app/placement-information/v1.0.0.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+  <head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>Sponsorship</title>
+  </head>
+  <body>
+    <h1>In-App Notification</h1>
+    <h2>Subject</h2>
+    <th:block th:fragment="subject">Placement Information</th:block>
+    <h2>Content</h2>
+    <th:block th:fragment="content">
+      <p>
+        For your placement starting
+        <b>
+          <th:block th:text="${startDate} ? |${#temporals.format(startDate, 'dd MMMM yyyy')}| : _">
+            (start date missing)
+          </th:block>
+        </b> at
+        <b>
+          <th:block th:text="${not #strings.isEmpty(site)} ? ${site} : _">
+            (site name missing)
+          </th:block>
+        </b> in
+        <b>
+          <th:block th:text="${not #strings.isEmpty(specialty)} ? ${specialty} : _">
+            (speciality name missing)
+          </th:block>
+        </b>, please see the below information.
+      </p>
+      <p>
+        The placement information reflects what we have recorded on our Training Information System (TIS) as
+        your next placement. <b>Please be aware that when your Grade, Whole Time Equivalent (WTE) or Post
+        Number changes, on TIS, a new placement will be created reflecting these changes.</b>
```

Comment:
"Number changes, on TIS," should be "Number changes on TIS,"

Reply:
I found the same sentence is being used in the `placem-updated-week-12`, I will update it.

Reply:
👌 I will leave it as is then.